pkg/ui/changelog/changelog.go

```diff
@@ -0,0 +1,64 @@
+package changelog
+
+import (
+	"errors"
+
+	tea "github.com/charmbracelet/bubbletea"
+	changelog "github.com/futurice/jalapeno/pkg/ui/changelog/prompt"
+)
+
+type Changelog struct {
+	Increment string
+	Msg       string
+}
+
+func RunChangelog() (Changelog, error) {
+	verInc, err := runSelectPrompt()
+
+	if err != nil {
+		return Changelog{}, errors.New("failed to get version type")
```

Comment:
The error from the prompt should be joined to this error
```suggestion
		return Changelog{}, fmt.Errorf("failed to get version type: %w", err)
```